puppet/zulip/files/postgresql/pg_backup_and_purge

```diff
@@ -53,7 +53,7 @@ with open('/var/lib/nagios_state/last_postgres_backup', 'w') as f:
     f.write(now.isoformat())
     f.write("\n")
 
-backups = {}  # type: Dict[datetime, str]
+backups: Dict[datetime, str] = {}
```

Comment:
I think we should exclude this line, since it's part of the `postgresql` rule.

Reply:
Anything worth excluding is worth testing: #14731.